modules/caddyhttp/reverseproxy/command.go

```diff
@@ -168,14 +170,17 @@ func cmdReverseProxy(fs caddycmd.Flags) (int, error) {
 	}
 
 	server := &caddyhttp.Server{
-		Routes: caddyhttp.RouteList{route},
-		Listen: []string{":" + fromAddr.Port},
+		Routes:    caddyhttp.RouteList{route},
+		Listen:    []string{":" + fromAddr.Port},
+		AutoHTTPS: &caddyhttp.AutoHTTPSConfig{},
```

Comment:
I don't think this is ideal -- one thing we tend to try to do is keep the generated config as lean as possible. So `AutoHTTPS` should only exist if it has any options set.

That means having a bit more code each place where the property is being touched that initializes the struct if it doesn't exist yet, with an if block. It's more verbose, but the output JSON is cleaner.